src/pages/uxp/reference-js/Global Members/HTML Elements/HTMLWebViewElement.md

```diff
@@ -30,6 +30,11 @@ Access to all the custom data attributes (data-*) set.
 ## innerText : `string`
 
 
+<a name="htmlelement-lang" id="htmlelement-lang"></a>
+
+## lang : `string`
```

Comment:
Yes! ✅ 🎉 

(not a change request, I'm just happy that I don't have to reference the infamous issue again)